domains/transfer/Transfer.API/API/Transfer/Api/v2023_01_01/Controllers/TransferAgreementHistoryEntriesController.cs

```diff
@@ -20,22 +21,27 @@ public class TransferAgreementHistoryEntriesController : ControllerBase
 
     public TransferAgreementHistoryEntriesController(ITransferAgreementHistoryEntryRepository historyEntryRepository) => this.historyEntryRepository = historyEntryRepository;
 
+    [Authorize(Policy = PolicyName.RequiresCompany)]
```

Comment:
What does this do exactly?

Reply:
It is a validation check, that assures, that each token, contains 3 claims:

OrganizationId, OrganizationName, Tin.

Reply:
Cool! Shouldn't it be on ClaimAutomationController as well then?